### PR TITLE
Matrix build for 5-10-15-20

### PR DESCRIPTION
**Pull request by yellowmegaman**

### CODE REVIEW

File: .github/workflows/loadtest.yaml [hunk 1]
```diff
@@ -32,187 +30,60 @@ jobs:
...
-          mkdir -p logs/${KUB_NAMESPACE}
-          kubectl -n ${KUB_NAMESPACE} logs --since=12h pulsar-0 > logs/${KUB_NAMESPACE}/pulsar-0.log
-          for pod in {0..19}; do
-            kubectl -n ${KUB_NAMESPACE} logs --since=12h virtual-$pod > logs/${KUB_NAMESPACE}/virtual-$pod.log
+          mkdir -p logs/${INSOLAR_NAMESPACE}
+          kubectl -n ${INSOLAR_NAMESPACE} logs --since=12h pulsar-0 > logs/${INSOLAR_NAMESPACE}/pulsar-0.log
+          for pod in $(kubectl -n $INSOLAR_NAMESPACE get po -lapp=virtual -o=custom-columns=NAME:.metadata.name --no-headers); do
+            kubectl -n ${INSOLAR_NAMESPACE} logs --since=12h $pod > logs/${INSOLAR_NAMESPACE}/$pod.log
           done
-      - name: Aggregate logs into a single archive
```
Comment (afiskon) [inline .github/workflows/loadtest.yaml:191]: Please keep this step

File: .github/workflows/loadtest.yaml [hunk 2]
```diff
@@ -32,187 +30,60 @@ jobs:
...
         if: always()
-        run: kubectl delete ns ${KUB_NAMESPACE} --ignore-not-found=true
-      - name: Send report
-        if: always()
-        run: |
-          REPORT_CSV_FILE=scaling.csv REPORT_PNG_FILE=report.png go run cmd/report/main.go
-          filename=$(date +"%F-%H-%M")-${GITHUB_SHA:0:8}.png
-          echo "::set-env name=FILE_NAME::${filename}"
-          curl -basic --user "${{ secrets.WEBDAV_USERNAME }}:${{ secrets.WEBDAV_PASSWORD }}" -T report.png "https://reports.garage.insolar.io/ve-perf-tests/${filename}"
-      - name: Send notification to Slack
```
Comment (afiskon) [inline .github/workflows/loadtest.yaml]: Ditto

**Review by afiskon — CHANGES_REQUESTED** [review]

**Review by dmitry-shidlovsky — APPROVED** [review]

File: .github/workflows/loadtest.yaml [hunk 3]
```diff
@@ -32,193 +30,96 @@ jobs:
...
           filename=$(date +"%F-%H-%M")-${GITHUB_SHA:0:8}.png
           echo "::set-env name=FILE_NAME::${filename}"
           curl -basic --user "${{ secrets.WEBDAV_USERNAME }}:${{ secrets.WEBDAV_PASSWORD }}" -T report.png "https://reports.garage.insolar.io/ve-perf-tests/${filename}"
-      - name: Upload logs to artifacts
-        if: always()
-        uses: actions/upload-artifact@v1
-        with:
-          name: nodes-logs
-          path: nodes-logs.tgz
-      - name: Upload scaling.csv to artifacts
```
Comment (afiskon) [inline .github/workflows/loadtest.yaml:210]: Please keep this.

File: .github/workflows/loadtest.yaml [hunk 4]
```diff
@@ -32,193 +30,96 @@ jobs:
...
-        if: always()
-        uses: actions/upload-artifact@v1
-        with:
-          name: scaling.csv
-          path: scaling.csv
-      - name: Upload report.png to artifacts
```
Comment (afiskon) [inline .github/workflows/loadtest.yaml]: And this too.

**Review by afiskon — CHANGES_REQUESTED** [review]

**Review by afiskon — APPROVED** [review]

**Pull request merged**